packages/EasyDoctrine/tests/Subscribers/EntityEventSubscriberTest.php

```diff
@@ -315,6 +315,95 @@ public function testEventsAreNotDispatchedWhenDispatcherIsDisabled(): void
         self::assertCount(0, $events);
     }
 
```

Comment:
What about covering also the following cases:
 - re-enabling: enable + disable(SomeEntity::class) + enable(SomeEntity::class)
 - re-disabling: disable + enable(SomeEntity::class) + disable(SomeEntity::class)
 - post-enabling: disable(SomeEntity::class) + enable
 - post-disabling: disable + enable(SomeEntity::class) + disable?

Reply:
The logic was simplified. After that only 3-rd point is actual and it was covered in test